pom.xml

```diff
@@ -12,14 +12,48 @@
         <jdk.version>11</jdk.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <maven.checkstyle.plugin.version>3.1.1</maven.checkstyle.plugin.version>
+        <maven.checkstyle.plugin.configLocation>
+            C:\Users\freak\IdeaProjects\hibernate-configuration-hw\checkstyle.xml
```

Comment:
The old way is also good  https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml